### DISCUSSION TITLE
Big Update: Background Images, Light and Dark Themes, Fixed Scrollbars, New CSS Classes, and More!

### DISCUSSION THREAD

**Discussion by Bluemoondragon07** [Bluemoondragon07/obsidian-big-and-bold#6] [Announcements]
![Screenshot 2023-03-23 212836](https://user-images.githubusercontent.com/121056178/227409007-f3f00ef5-ad80-4b6c-bbf3-594b41ec38dd.png)

Many exciting features have been added to Big & Bold in this new update!

# Style Settings
1. There are now separate themes for light and dark mode.
    1. New Color Scheme: **Electric**
    2. Spectre has become dark-theme only.
    3. Library has become light-theme only.
4. You can now set a background image for your workspace! There are 25 images to choose from, all selected with care from Unsplash and a few wallpaper websites. 
5. Option to hide frontmatter in all notes
6. Option to hide inline title in all notes
7. New 'Minimalistic Frontmatter' option.
8. Inline titles now have their own special options.
10. A few fonts that I decided were too unreadable (such as Fleur de Leah and Sail) have been replaced with new, cleaner-looking fonts. Sorry, I just can't stop adding fonts!
11. Added page gallery options.
12. Many other additions that aren't that important.

# New CSS Classes
1. You can hide frontmatter in specific notes with the css class `hide-m`
3. You can hide inline titles in specific notes with the css class `hide-title`

# New Image Alt
1. Use the `no-shadow` alt to remove the box shadow on an image. This also works on Excalidraw embeddings!

# Fixes
1. Fixed the gap under the sidebar button when page shadow or floating document settings are turned on.
2. Scrollbars should now be hidden on Mac devices.

# Small Changes that Probably Don't Matter
1. I changed the way tabs behave when **Full Focus** is turned on. Now, they appear on hover, unless there is only a single tab. If there's a single tab, that single tab is hidden--kind of like in Shimmering Focus.
2. Modal animation (affects command palette and quick switcher)
4. Colorful Headings look a bit different in the default color scheme.

<hr /><em>This discussion was created from the release <a href='https://github.com/Bluemoondragon07/obsidian-big-and-bold/releases/tag/v0.2.8'>Big Update: Background Images, Light and Dark Themes, Fixed Scrollbars, New CSS Classes, and More!</a>.</em>

**Comment by yarub123** [comment]
Such a delicious update. Thank you for your hard work, truly commendable ❤️

Also before I forget, how does one make a page gallery? Should we just use the page gallery plugin for that?

**Reply by Bluemoondragon07** [reply]
Thank you, I'm glad you like the update! You do need the Page Gallery plugin to make a page gallery. I like to use this codeblock for page galleries because it gives me all the notes in the current folder:

````yaml
```page-gallery
from: '"/"'
where: 'file.folder = this.file.folder and file.name != this.file.name'
fields: [file.name, file.ctime]
mode: image
```
````

Try it out! 😀

**Reply by yarub123** [reply]
You're delicious. Thank you haha 🙏
